src/preload/syscallbuf.c

```diff
@@ -194,6 +194,18 @@ static void local_memcpy(void* dest, const void* source, int n) {
                        : "+S"(source), "+D"(dest), "+c"(n)
                        :
                        : "cc", "memory");
+#elif defined(__aarch64__)
+  long c;
+  __asm__ __volatile__("cbz %3, 2f\n"
```

Comment:
Do we need something faster here and link in the standard assembly memcpy routine?